src/org/labkey/test/util/EscapeUtil.java

```diff
@@ -149,4 +149,32 @@ public static String getTextChoiceValidatorExpression(List<String> options)
                 .map(value -> value.replaceAll("([\\\\|])", "\\\\$1"))
                 .collect(Collectors.joining("|"));
     }
+
+    /**
+     * Takes an identifier (a field or domain value) and enquotes it, while also escaping any quotes
+     * it might have.  This his helpful for using randomly-generated identifiers in sql queries.
+     * @param value
+     * @return
```

Comment:
These aren't adding any value. I suggest deleting them, and for the other new method.